<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QVTKOpenGLWidget" name="vtkWidget" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Axes</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_A1">
                  <property name="text">
                   <string>A1</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A1">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_A2">
                  <property name="text">
                   <string>A2</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A2">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_A3">
                  <property name="text">
                   <string>A3</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A3">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_A4">
                  <property name="text">
                   <string>A4</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A4">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_A5">
                  <property name="text">
                   <string>A5</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A5">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_A6">
                  <property name="text">
                   <string>A6</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A6">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>End effector</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_X">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_X">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_Y">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_Y">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_Z">
                  <property name="text">
                   <string>Z</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_Z">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_A">
                  <property name="text">
                   <string>A</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_A">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_B">
                  <property name="text">
                   <string>B</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_B">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_C">
                  <property name="text">
                   <string>C</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox_C">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-1000000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>440</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
